cloud/deployment/getting-started.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Getting Started
+---
+
+## Get the image
+
+1. After getting an issued Docker PAT
+
+2. Authenticate with your token:
+```shell
+docker login --username chainlit 
+# When prompted for a password, use the token
+```
+
+3. Pull the image
+```shell
+docker pull chainlit/chainlit-cloud.<version>
```

Comment:
would say latest here?

Reply:
For sure, but in that case we need to handle the latest tag:
<img width="953" alt="Screenshot 2023-09-29 at 09 34 46" src="https://github.com/Chainlit/docs/assets/10745953/cc2bc54b-8295-47ad-8160-75a02ec56d25">